src/main/java/com/exasol/adapter/response/CreateVirtualSchemaResponse.java

```diff
@@ -0,0 +1,59 @@
+package com.exasol.adapter.response;
+
+import com.exasol.adapter.metadata.SchemaMetadata;
+import com.exasol.adapter.response.converter.ResponseJsonConverter;
+
+/**
+ * Used in overloaded convert() method of {@link ResponseJsonConverter}
```

Comment:
"This class is an abstract representation of a response created by a Virtual Schema Adapter as result of a request to create a new virtual schema."

Reply:
fixed